### PR TITLE
T1-6 bugfix something

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: dddd [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function test() 
+{
+    console.log()
```
Comment (mpaz-redstage) [inline dddd:3]: Remove console.log()

Comment (rpfortes-redstage) [reply dddd:3]: all good

Comment (psantos-redstage) [reply dddd:3]: test

**Review by psantos-redstage — APPROVED** [review]

File: dddd (hunk 1 above)
Comment (rpfortes-redstage) [reply dddd:3]: asd

**Review by rpfortes-redstage — APPROVED** [review]
ok ok

**Pull request closed without merging**